india_banking/india_banking/doc_events/bank_account/bank_account.py

```
@@ -77,3 +85,18 @@ def update_party_transaction_currency(doc):
 		) or frappe.db.get_value("Company", doc.company, "default_currency")
 	elif doc.is_company_account:
 		doc.currency = frappe.db.get_value("Company", doc.company, "default_currency")
+
+
+def get_bank_details_from_branch_code(branch_code):
+	url = f"https://ifsc.razorpay.com/{branch_code}"
+
+	try:
+		response = requests.get(url)
```

Ruff (0.15.2)

[error] Probable use of `requests` call without timeout

(S113)